src/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,29 @@
+import { ReactNode } from "react"
+import { Status } from "./types"
+import styles from "./Toast.module.css"
+
+interface ToastProps {
+  status?: Status
+  message: string
+  action?: ReactNode
+}
+
+export function Toast({ action, message, status = "success" }: ToastProps) {
+  return (
+    <div
+      data-testid={`toast_${message}`}
+      role="alert"
+      aria-live={status === "error" ? "assertive" : "polite"}
+      aria-label={`a ${status} toast`}
+      data-type={status}
+      className={styles.toast}
+    >
+      <div className={styles.toastContent}>
+        <div>{message}</div>
+        {action ? (
+          <div className={styles.toastActionContainer}>{action}</div>
+        ) : null}
```

Comment:
I think I've just gotten into this habit because I have had issues where truthy values make it through and render as text on the screen, but maybe that speaks more of my mishandling of truthy values.